src/px-script/main/nav/index.js

```diff
@@ -104,29 +106,29 @@ class Nav {
     }
 }
 
-const nav = (() => {
-    const init = () => {
-        let navs = document.querySelectorAll(".nav");
+const init = ids => {
+    let navs = ids || ids === "" ? getElementsByIds(ids, "nav") : document.querySelectorAll(".nav");
```

Comment:
Consider creating a helper method to do this as it is repeated several places. 🛩

Reply:
As most of the logic depends on whether or not i get sent an argument I don't think there are a lot of good ways to simplify this.